ctapipe/io/hdf5tableio.py

```diff
@@ -105,6 +116,12 @@ def close(self):
 
         self._h5file.close()
 
+    def build_prefix(self, container):
```

Comment:
Shouldn't this prefix-building code go in the base class? I would assume it's the same idea for any output type (e.g. FITS or other table format).  This way, the prefixes will only work with HDF5 outputs.  Of course we haven't implemented any other TableWriters, but I intend to do at least an AstropyTableWriter and a pure FITSTableWriter

Reply:
yes, good point